hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/impl/CassAlertsServiceImpl.java

```diff
@@ -1143,12 +1149,12 @@ public Event getEvent(String tenantId, String eventId, boolean thin) throws Exce
             List<Event> ordered = events;
             if (pager.getOrder() != null) {
                 pager.getOrder()
-                        .stream()
-                        .filter(o -> o.getField() != null && o.getDirection() != null)
-                        .forEach(o -> {
-                            EventComparator comparator = new EventComparator(o.getField(), o.getDirection());
-                            Collections.sort(ordered, comparator);
-                        });
+                .stream()
```

Comment:
Yes, it's annoying to get the settings just right in Eclipse.  I'll recheck my settings, I don't want it to reformat the entire file.  Sometimes I think I accidentally format the file myself and don't realize until it's time to commit.